sdks/python/setup.py

```diff
@@ -444,7 +444,7 @@ def get_portability_package_data():
               'google-cloud-bigquery-storage>=2.6.3,<3',
               'google-cloud-core>=2.0.0,<3',
               'google-cloud-bigtable>=2.19.0,<3',
-              'google-cloud-spanner>=3.0.0,<4',
+              'google-cloud-spanner>=3.0.0,<3.48',
```

Comment:
@Polber @reeba212 catching up on this, which suites were failing such that we needed to add this? I'd like to follow up with the spanner folks to make sure they're aware of the breaking change

Reply:
There were two checks that were failing and both had the same underlying error: 
`ModuleNotFoundError: No module named 'grpc_interceptor'`